config/config.go

```diff
@@ -180,3 +182,9 @@ func Validate(conf *Config) error {
 	}
 	return nil
 }
+var version = "dev build <no version set>"
+
+func PrintVersion() {
+	fmt.Printf("%s %s\n", filepath.Base(os.Args[0]), version)
+	os.Exit(0)
+}
```

Comment:
I don't think this belongs here. This is specifically a "main" concern so it should stay there. This was added to enable printing versions of the "hauser" binary once we start building them.

Reply:
I was try to deduplicate the code, since other instances of "main" might want to print version of code also.

I moved it back to main.  Fixed.